lib/client/index.ts

```diff
@@ -280,8 +286,10 @@ export class LibraClient {
           new LibraTransactionResponse(
             new LibraSignedTransaction(transaction, sender.keyPair.getPublicKey(), senderSignature),
             response.getValidatorId_asU8(),
-            response.getAcStatus(),
-            response.getMempoolStatus(),
+            response.getAcStatus() === undefined ? -1 : (response.getAcStatus() as AdmissionControlStatus).getCode(),
```

Comment:
Instead of `response.getActStatus() === undefined` use `response.hasAcStatus()`,
Then update the `LibraAdmissionControlStatus` enum to have a -1 value to account for the -1 default value. The name of the -1 value can be `UNKNOWN`.

Reply:
Ok,I will do it.